sktime/tests/test_resizing.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import pandas as pd
+
+from sktime.datasets import load_basic_motions
+from sktime.transformers.resizing import TSResizeTransform
+
+# load data
+df, _ = load_basic_motions(split='TRAIN', return_X_y=True)
+
+
+def cut_df_ts(df):
+    for row_i in range(df.shape[0]):
+        for dim_i in range(df.shape[1]):
+            ts = df.at[row_i, f'dim_{dim_i}']
+            df.at[row_i, f'dim_{dim_i}'] = pd.Series(ts.tolist()[:len(ts)-dim_i-1])
+
+
+
+def test_resizing():
```

Comment:
this looks good, could you expand a little bit on the comments to explain how you're testing the transformer (just a few more words would already be quite helpful)? or making the code a bit more readable :-)

Reply:
done